<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>656</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item alignment="Qt::AlignTop">
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="titleText">
         <property name="font">
          <font>
           <pointsize>24</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Welcome</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="lyceumLogo">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>assets/lyceumLogo.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
         <property name="openExternalLinks">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="Wrapper">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item alignment="Qt::AlignVCenter">
        <widget class="QFrame" name="formFrame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="4" column="0">
           <widget class="QLabel" name="nameLabel">
            <property name="text">
             <string>Name</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QFrame" name="nameTextBox">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLineEdit" name="lastName">
               <property name="placeholderText">
                <string>Last</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="firstName">
               <property name="placeholderText">
                <string>First</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="middleInitial">
               <property name="placeholderText">
                <string>Middle</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="ageLabel">
            <property name="text">
             <string>Age</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="age"/>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="birthDateLabel">
            <property name="text">
             <string>Birth date</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QFrame" name="birthDateTextBox">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QComboBox" name="month">
               <property name="mouseTracking">
                <bool>false</bool>
               </property>
               <property name="editable">
                <bool>false</bool>
               </property>
               <property name="maxVisibleItems">
                <number>12</number>
               </property>
               <property name="minimumContentsLength">
                <number>0</number>
               </property>
               <property name="placeholderText">
                <string>Month</string>
               </property>
               <item>
                <property name="text">
                 <string>January</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Febuary</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>March</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>April</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>May</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>June</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>July</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>August</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>September</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>October</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>November</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>December</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="day">
               <property name="placeholderText">
                <string>day</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="year">
               <property name="placeholderText">
                <string>year</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="genderLabel">
            <property name="text">
             <string>Sex</string>
            </property>
           </widget>
          </item>
          <item row="10" column="0">
           <widget class="QLabel" name="yearCourseSection">
            <property name="text">
             <string>Year/Course/Section</string>
            </property>
           </widget>
          </item>
          <item row="10" column="1">
           <widget class="QFrame" name="frame_2">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLineEdit" name="academicYear">
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="course">
               <property name="placeholderText">
                <string>Select</string>
               </property>
               <item>
                <property name="text">
                 <string>Course 1</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Course 2</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="section">
               <property name="placeholderText">
                <string>Select</string>
               </property>
               <item>
                <property name="text">
                 <string>Section 1</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Section 2</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="11" column="0">
           <widget class="QLabel" name="gNameLabel">
            <property name="text">
             <string>Guardian's Name</string>
            </property>
           </widget>
          </item>
          <item row="11" column="1">
           <widget class="QLineEdit" name="gName">
            <property name="placeholderText">
             <string>Type here...</string>
            </property>
           </widget>
          </item>
          <item row="12" column="0">
           <widget class="QLabel" name="gAddressLabel">
            <property name="text">
             <string>Guardian's Address</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QLineEdit" name="contact">
            <property name="placeholderText">
             <string>09XXXXXXXXX</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Contact #</string>
            </property>
           </widget>
          </item>
          <item row="12" column="1">
           <widget class="QTextEdit" name="gAddress"/>
          </item>
          <item row="8" column="1">
           <widget class="QComboBox" name="sex">
            <property name="mouseTracking">
             <bool>false</bool>
            </property>
            <property name="editable">
             <bool>false</bool>
            </property>
            <property name="currentText">
             <string/>
            </property>
            <property name="maxVisibleItems">
             <number>12</number>
            </property>
            <property name="minimumContentsLength">
             <number>0</number>
            </property>
            <property name="placeholderText">
             <string>Select</string>
            </property>
            <item>
             <property name="text">
              <string>Male</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Female</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item alignment="Qt::AlignBottom">
     <widget class="QGroupBox" name="navigation">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="title">
       <string/>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="next">
         <property name="text">
          <string>Next</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="reset">
         <property name="text">
          <string>Reset</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="back">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Back</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
